server/src/test/java/org/opensearch/rest/action/cat/RestShardsActionTests.java

```diff
@@ -76,10 +79,15 @@ public void testBuildTable() {
             Path path = createTempDir().resolve("indices")
                 .resolve(shardRouting.shardId().getIndex().getUUID())
                 .resolve(String.valueOf(shardRouting.shardId().id()));
+            CommonStats commonStats = mock(CommonStats.class);
+            DocsStats docsStats = mock(DocsStats.class);
+            when(docsStats.getCount()).thenReturn(numDocs);
+            when(docsStats.getDeleted()).thenReturn(numDeletedDocs);
+            when(commonStats.getDocs()).thenReturn(docsStats);
```

Comment:
These look like plain Java classes. Do you need to mock them as opposed to doing the following?

```
CommonStats commonStats = new CommonStats();
commonStats.docs = new DocsStats(numDocs, numDeletedDocs, 0);
```